packages/typespec-ts/test/modularUnit/modelsGenerator.spec.ts

```diff
@@ -804,4 +804,83 @@ describe("inheritance & polymorphism", () => {
       }`
     );
   });
+
+  it.only("should handle inheritance model in operations", async () => {
```

Comment:
skip `only` & could you add test cases for A -> B -> C with different discriminator?